src/utilities/display/demo/hidden-visually.twig

```diff
@@ -1,6 +1,6 @@
 <a href="https://github.com/cloudfour/cloudfour.com-patterns">
-  {% include '../../../assets/icons/github.svg.twig' with { class: 'c-icon c-icon--inline' } %}
-  <span class="u-hidden-visually">
-    View this project on GitHub
-  </span>
+	{% include '../../../assets/icons/brands/github.svg.twig' with { class: 'c-icon c-icon--inline' } %}
+	<span class="u-hidden-visually">
+		View this project on GitHub
+	</span>
```

Comment:
Wrong tab format. Do you have the editor config plugin installed?

Reply:
> Do you have the editor config plugin installed?

I do, but I ended up disabling it for this particular file because I was experiencing that issue where I was getting spaces → tabs and extra `</p>` tags. 

Fixing...